mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -1861,6 +1901,98 @@ resource "google_project_iam_member" "composer-worker" {
 `, environment, network, subnetwork, serviceAccount)
 }
 
+func testAccComposerEnvironment_airflow2RecoveryCfg(name, network, subnetwork string) string {
+	return fmt.Sprintf(`
+resource "google_composer_environment" "test" {
+  name   = "%s"
+  region = "us-central1"
+
+  config {
+    node_config {
+      network          = google_compute_network.test.self_link
+      subnetwork       = google_compute_subnetwork.test.self_link
+      ip_allocation_policy {
+        cluster_ipv4_cidr_block = "10.0.0.0/16"
+      }
+    }
+
+    software_config {
+      image_version = "composer-2-airflow-2"
+    }
+
+    recovery_config {
+			scheduled_snapshots_config {
+				enabled = 									 true
+				snapshot_location = 				 "gs://example-bucket/environment_snapshots"
+				snapshot_creation_schedule = "0 4 * * *"
+				time_zone = 								 "UTC+01"
+			}
```

Comment:
weird indentations

Reply:
Yeah, the indentation is bit off here. We use spaces instead of tabs for resource configuration. Would you mind cleaning it up here? Thanks!

Reply:
Fixed